prow/spyglass/lenses/buildlog/lens.go

```diff
@@ -46,13 +49,26 @@ const (
 )
 
 type config struct {
-	HighlightRegexes []string `json:"highlight_regexes"`
-	HideRawLog       bool     `json:"hide_raw_log,omitempty"`
+	HighlightRegexes []string         `json:"highlight_regexes"`
+	HideRawLog       bool             `json:"hide_raw_log,omitempty"`
+	Highlighter      *highlightConfig `json:"highlighter"`
+}
+
+type highlightConfig struct {
+	// Endpoint specifies the URL to send highlight requests
+	Endpoint string `json:"endpoint"`
+	// Pin should automatically save the highlight when set.
+	Pin bool `json:"pin"`
+	// Overwrite should replace any existing highlight when set.
+	Overwrite bool `json:"overwrite"`
+	// Auto should request highlights before loading the page, implies Pin.
```

Comment:
Is `auto && !pin` validated?

Reply:
Yes, see `auto pin` in `TestAnalyzeArtifact`